contributing.md

```diff
@@ -0,0 +1,106 @@
+# Enarx RFCs
+
+## Contributing
+
+### Do you need an RFC?
+
+Use an RFC to advocate substantial changes to the Aries ecosystem, where
+those changes need to be understood by developers who *use* Aries. Minor
+changes are not RFC-worthy, and changes that are internal in nature,
+invisible to those consuming Aries, should be documented elsewhere.
+
+### Preparation
+
+Before writing an RFC, consider exploring the idea on the
+[chat](https://gitter.im/enarx/community), on a daily meeting call
+(see the [wiki page on contributing](
+https://github.com/enarx/enarx/wiki/How-to-contribute),
+Encouraging feedback from maintainers is a good sign that you're on
+the right track.
+
+### How to propose an RFC
+
+  - Fork [the RFC repo](https://github.com/enarx/rfc).
+  - Pick a descriptive folder name for your RFC. Don't pick a number yet.
+  - Decide which parent folder is appropriate for your RFC.
+    If it is about a specific protocol or decorator or feature, its parent
+    should be /features; if it is about a concept that will be used in many
+    different features, its parent should be /concepts.
+  - Create the folder and copy `0000-template.md` to `<parent>/<your folder name>/README.md`.
+  - Fill in the RFC. [Use MUST and SHOULD per standard conventions](https://tools.ietf.org/html/rfc2119). Put care into the details:
+    RFCs that do not present convincing motivation, demonstrate an understanding of the impact of the
+    design, or are disingenuous about the drawbacks or alternatives tend to be
+    poorly received. You can add supporting artifacts, such as diagrams and sample
+    data, in the RFC's folder.
+  - Consider how the RFC should be [tagged](/tags.md).
+  - Assign a number to your RFC. Get the number by inspecting open and closed PRs against
+    this repo to figure out what the next PR number will be). Rename your folder from
+    `<your folder name>` to `<your 4-digit number>-<your folder name>`. At the
+    top of your README.md, modify the title so it is in the form: `<your 4-digit
+    number>: Friendly Version of Your Title`. Commit your changes.
+  - Commit the updated version of /index.md and push your changes.
+  - Submit a pull request.
```

Comment:
If anything, I think this is where people would get lost or do something incorrectly; wondering if there is any way to make this more streamlined / intuitive and not require so many instructions (although there may not be). At least we can always request changes on a PR.

Reply:
Related thought: is it feasible to have some type of CI tests for this whole process to make sure people did do it correctly?

Reply:
This feels too complex.  How about modelling on the Rust template?  https://github.com/rust-lang/rfcs/blob/master/0000-template.md

Reply:
It does feel too complex, i agree. 

`contributing.md` is supposed to be the file that helps you fill in the template file (0000-template.md). Rust doesn't use it though, only Aries, but i've tried to simplify it.